.teamcity/settings.kts

```diff
@@ -59,8 +57,6 @@ project {
             RelativeId("E2E"),
             RelativeId("SmokeTests"),
             RelativeId("Frontpage"),
-            RelativeId("Docs"),
-            RelativeId("Lint"),
```

Comment:
I remove these, because they are both no longer needed (docs no longer exists) (linting errors when it's successful in other CI)